src/js/background.js

```diff
@@ -206,9 +217,36 @@ chrome.tabs.onUpdated.addListener((tabId, changeInfo, tab) => {
 })
 
 chrome.tabs.onActivated.addListener((tabId, windowId) => {
+  if (isFirefox) {
```

Comment:
I do not think this code block is necessary. I believe it is covered by the update code block.